data/mapcss-mapping.csv

```diff
@@ -1683,3 +1683,4 @@ railway|station|subway|singapore;[railway=station][transport=subway][city=singap
 railway|subway_entrance|singapore;[railway=subway_entrance][city=singapore];;name;int_name;1595;
 railway|station|subway|boston;[railway=station][transport=subway][city=boston],[railway=station][station=subway][city=boston];;name;int_name;1596;
 railway|subway_entrance|boston;[railway=subway_entrance][city=boston];;name;int_name;1597;
+railway|station|overground|london;[railway=station][operator=London Overground];;name;int_name;1598;
```

Comment:
Well, if it works good (probably should add one test in osm_tyoe_test.cpp) I'm ok with that.

@RedAuburn Can we trust operator=London Overground ?

Reply:
hmmm... looks like it's gonna be a bit more complicated :/

this query finds all 113 stations the Overground serves:
```
[out:json][timeout:25];
{{geocodeArea:United Kingdom}}->.searchArea;
// OR
(
  // should match =London Underground;London Overground
  nwr["railway"="station"]["network"~"London Overground"](area.searchArea);
  nwr["railway"="station"]["operator"~"London Overground"](area.searchArea);
);
// print results
out geom;
```

need to search for all nodes where _either_ `network=*` or `operator=*` _contains_ `London Overground` (non-case sensitive). And not just an exact string match as POI can have things like `network=London Underground;London Overground`

Reply:
`railway|station|overground|london;[railway=station][operator=London Overground],[railway=station][network=London Overground];` :)

but IDK how do we treat case and multiple values might need a special processing...

Reply:
Would adding another wildcard to signify case insensitivity not be ok? I'd like to help make the changes for how this might be parsed, but I'm having trouble working out how to build and run the generator code by itself (and not driven by tools/python/map_generator) to allow me to debug it.

Reply:
IMO all OSM tags matching should be case-insensitive (maybe its the case already?)

Reply:
> but IDK how do we treat case and multiple values might need a special processing...
@vng do you know if the generator can handle case & multiple values?

Reply:
If have doubts, better make tests like in osm_type_test.cpp